content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -715,6 +716,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `TopologyManager`: Enable a mechanism to coordinate fine-grained hardware resource
   assignments for different components in Kubernetes. See
   [Control Topology Management Policies on a node](/docs/tasks/administer-cluster/topology-manager/).
+  Topology Manager graduated to GA in 1.27 and is enabled by default. If you are using a version of
+  Kubernetes older than v1.27, switch to the documentation for that version of Kubernetes.
```

Comment:
This two lines can be safely dropped.

Reply:
@swatisehgal Please drop the two lines when you have a moment and we can merge this PR.